website/content/docs/v0.3/Guides/sidero-on-rpi4.md

```diff
@@ -0,0 +1,156 @@
+---
+description: "Running Sidero on Raspberry Pi 4 to provision bare-metal servers."
+title: Sidero on Raspberry Pi 4
+weight: 7
+---
+
+Sidero doesn't require lots of computing resources, so SBCs might be a perfect fit to run
+the Sidero management cluster.
+In this guide, we are going to install Talos on Raspberry Pi4, deploy Sidero and other CAPI components.
+
+## Prerequisites
+
+Please see Talos documentation for additional information on [installing Talos on Raspberry Pi4](https://www.talos.dev/docs/v0.10/single-board-computers/rpi_4/).
+
+Download `clusterctl` CLI executable version at least 0.3.17 from [CAPI releases](https://github.com/kubernetes-sigs/cluster-api/releases).
+
+## Installing Talos
+
+Prepare SD card with Talos image, boot RPi4 with the SD card.
+Talos should drop into maintenance mode printing the acquired IP address.
+Record the IP address as the environment variable `SIDERO_ENDPOINT`:
+
+```bash
+export SIDERO_ENDPOINT=192.168.x.x
+```
+
+> Note: it makes sense to transform DHCP lease for RPi4 into a static reservation so that RPi4 always has the same IP address.
```

Comment:
Another option that I thought was pretty neat, was to use a VIP. This works too, and not sure I have a strong preference for documenting/recommending one way over another.

Reply:
VIP still needs a reservation though on DHCP side?